### ISSUE TITLE
preemptive Ransac

### ISSUE THREAD

**Issue by LiliMeng** [ISUE/relocforests#10]
Hi,

      did you use preemptive ransac for this code? thanks a lot!

**Comment by connerbrooks** [comment]
Yes, the RANSAC implementation is the same from _Scene Coordinate Regression Forests for Camera Relocalization in RGB-D Images_, which uses an adapted preemptive RANSAC (Algorithm 1).

This code is a direct implementation of that paper.
